templates/closingdays/index.html.twig

```diff
@@ -0,0 +1,96 @@
+{# closingdays/index.html.twig #}
+
+{% extends 'base.html.twig' %}
+
+{% block page %}
+  <div id='joursFeries'>
+    <h3>Jours fériés et jours de fermeture</h3>
+    <form name='form1' method='get' action='{{ asset("closingday") }}'>
+      <input type='hidden' name='page' value='{{ asset("closingday") }}' />
+      Sélectionnez l'année à paramétrer
+      <select name='annee' onchange='document.form1.submit();'>
+        <option value=''>&nbsp;</option>
+        {% for y in years %}
+          {% if selectedYears[loop.index0] is not null %}
```

Comment:
You have a variable selectYear. A more simple way to do this should be:
{% if y == selectedYear %}

So the variable selectYears is now useless

Reply:
There should be several years, that's the reason why I use an array. I will check this issue for sure.

Reply:
The edition didn't affect the display, thanks !